main_test.go

```diff
@@ -5,17 +5,15 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/carlogit/phash"
+	"github.com/liamg/aminal/config"
+	"github.com/liamg/aminal/gui"
+	"github.com/liamg/aminal/terminal"
```

Comment:
conventional Go practice is to group imports into 3 sections:
* standard library
* third party
* local

Reply:
done in https://github.com/liamg/aminal/pull/262/commits/29e2f6861a25f160506bded5cce4e48e6e836a80